bin/publish

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
If we did this based on a tag being pushed, that'd get rid of the need to call `bin/publish` - Right?